src/if/os_interface_lag.cpp

```diff
@@ -91,10 +92,12 @@ bool INTERFACE::os_interface_lag_attrs_handler(if_details *details, cps_api_obje
 
     if (details->_info_kind!=nullptr && !strncmp(details->_info_kind, "tun", 3)) {
          if(details->_attrs[IFLA_MASTER]!=NULL  && ((details->_flags & IFF_SLAVE)!=0)) {
-            EV_LOG(INFO, NAS_OS,3, "NET-MAIN", "Received tun %d and state 0x%x",
+             EV_LOGGING(NAS_OS, INFO, "NET-MAIN", "Received tun %d and state 0x%x",
```

Comment:
The indentations are bit off here. Plz check the previous code.

Reply:
it's fine.